tests/integration/mutation/update/crdt/pcounter_test.go

```diff
@@ -123,6 +123,12 @@ func TestPCounterUpdate_IntKindWithPositiveIncrement_ShouldIncrement(t *testing.
 func TestPCounterUpdate_IntKindWithPositiveIncrementOverflow_RollsOverToMinInt64(t *testing.T) {
 	test := testUtils.TestCase{
 		Description: "Positive increments of a P Counter with Int type causing overflow behaviour",
+		SupportedClientTypes: immutable.Some([]testUtils.ClientType{
+			// JS client does not support 64 bit integers
```

Comment:
question: Why not?

Reply:
Can you add/document this somewhere incase there are others who wonder about this later.